website/docs/commands/fmt.md

```diff
@@ -167,3 +169,7 @@ If the `--save-scalafmt-conf` option is passed, then `fmt` command behaves as fo
 - In the **first** case `fmt` uses the found `.scalafmt.conf` file to run `scalafmt`.
 - In the **second** case `fmt` [infers](/docs/commands/fmt#scalafmt-version-and-dialect) missing parameters, writes them directly into the previously found file and then uses this file to run `scalafmt`.
 - In the **third** case `fmt` creates a `.scalafmt.conf` file in the current workspace directory, writes [inferred](/docs/commands/fmt#scalafmt-version-and-dialect) version and dialect into it and uses it to run `scalafmt`.
+
+:::note
+If the configuration is passed in the `--scalafmt-conf-str` option, Scala CLI behaves exactly the same as it would find `.scalafmt.conf` with specified configuration in the workspace.
```

Comment:
```suggestion
If the configuration is passed in the `--scalafmt-conf-str` option, Scala CLI will behave exactly the same as if it would find the specified configuration in a `.scalafmt.conf` file in the workspace.
```